Analysis/Tasks/PWGHF/taskLcK0sp.cxx

```diff
@@ -0,0 +1,157 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskLcK0sp.cxx
+/// \brief LcK0sp analysis task
+///
+/// \author Chiara Zampolli, <Chiara.Zampolli@cern.ch>, CERN
+///
+/// based on taskD0.cxx, taskLc.cxx
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "AnalysisDataModel/HFSecondaryVertex.h"
+#include "AnalysisDataModel/HFCandidateSelectionTables.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::aod::hf_cand_casc;
+using namespace o2::framework::expressions;
+
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  ConfigParamSpec optionDoMC{"doMC", VariantType::Bool, false, {"Fill MC histograms."}};
+  workflowOptions.push_back(optionDoMC);
+}
+
+#include "Framework/runDataProcessing.h"
+
+/// LcK0sp analysis task
+struct TaskLcK0sp {
```

Comment:
`TaskLcToK0sP`

Reply:
But the symbol of the proton is small "p". I understand that you (or O2?) have the camelCase rule, but I would leave the proton small. No?

Reply:
done, better to follow the rules.